packages/cli/test/fixtures/discover/index.js

```diff
@@ -0,0 +1,19 @@
+const fs = require('fs');
+
+exports.SANDBOX_FILES = [
+  /*  {
+    path: '',
+    file: 'test-schema-defs.json',
+    content: fs.readFileSync(require.resolve('./test-schema-defs.json')),
+  },*/
+  /*  {
+    path: 'dist/src/datasources',
+    file: 'mem.datasource.json',
+    content: fs.readFileSync(require.resolve('./mem.datasource.json')),
+  },*/
```

Comment:
Please remove commented-out code. Also if `test-schema-defs.json` and `mem.datasource.json` are no longer used, then remove those files too.

Reply:
done